test-data/unit/check-modules.test

```diff
@@ -1641,6 +1641,40 @@ m = n  # E: Cannot assign multiple modules to name 'm' without explicit 'types.M
 
 [builtins fixtures/module.pyi]
 
+[case testModuleAliasToQualifiedImport]
+import os.path
```

Comment:
I wonder if picking a different package.module name than os.path makes more sense here? I was confused by the whatever() call until I realized this has nothing to do with the stdlib os.path.